components/Main.vue

```diff
@@ -157,9 +183,10 @@ export default {
     data() {
         return {
             categories: [],
-            searchText: 'Alerts',
+            // searchText: "Alerts",
+            searchText: "Pricings",
```

Comment:
Back a search text to alerts because this will be in production after its merge.

Reply:
yes sorry